test/e2e/framework/util.go

```diff
@@ -2206,6 +2205,7 @@ func (b kubectlBuilder) Exec() (string, error) {
 		return "", fmt.Errorf("timed out waiting for command %v:\nCommand stdout:\n%v\nstderr:\n%v\n", cmd, cmd.Stdout, cmd.Stderr)
 	}
 	Logf("stderr: %q", stderr.String())
+	Logf("stdout: %q", stdout.String())
```

Comment:
I am not sure about this line. Could you please clarify why you need it here and not in its original place? I wonder if it changes anything w.r.t. these way logs are parsed for display in test grid.

@krzyzacy could you please take a look or point to someone who might be able to advice here?

Reply:
I have no clue here... looks like @janetkuo wrote the timeout logic here in https://github.com/nikhiljindal/kubernetes/commit/13718846d8fd33d7543c448ac2d33184e25431d4?

Reply:
I changed it here so that stdout is printed for all callers. We were not consistent before this change.

I dont expect this to change the way logs are parsed by testgrid. @krzyzacy can confirm

Reply:
yeah testgrid does not parsing log output

Reply:
Thanks for confirming @krzyzacy